test/jdk/javax/swing/border/Test4247606.java

```diff
@@ -21,36 +21,55 @@
  * questions.
  */
 
-/*
- * @test
- * @bug 4247606
- * @summary BorderedPane appears wrong with Title Position Below Bottom
- * @author Andrey Pikalev
- * @run applet/manual=yesno Test4247606.html
- */
-
 import java.awt.BorderLayout;
 import java.awt.Color;
+import java.awt.Dimension;
 import javax.swing.BorderFactory;
-import javax.swing.JApplet;
 import javax.swing.JButton;
 import javax.swing.JComponent;
 import javax.swing.JPanel;
 import javax.swing.border.Border;
 import javax.swing.border.TitledBorder;
 
-public class Test4247606 extends JApplet {
-    public void init() {
-        JButton button = new JButton("Button"); // NON-NLS: the button text
+/*
+ * @test
+ * @bug 4247606
+ * @summary BorderedPane appears wrong with Title Position Below Bottom
```

Comment:
Minor suggestion else looks good to me.

```suggestion
 * @summary BorderedPane appears wrong with title position below bottom
```